Bottom: 42a2491d83ba041a57114b79228b5b05ff5e7581
Top:    6d38dd3a1877b6c751e77e4fddeb3f76e6a66c0a
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-02 16:50:55 +0800

Refresh of autofs.debug

---

diff --git a/modules/lookup_nisplus.c b/modules/lookup_nisplus.c
index b3b44c1..fc31af3 100644
--- a/modules/lookup_nisplus.c
+++ b/modules/lookup_nisplus.c
@@ -609,8 +609,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 	debug(ap->logopt, MODPREFIX "%s -> %s", key, mapent);
 	ret = ctxt->parse->parse_mount(ap, key, key_len,
 				       mapent, ctxt->parse->context);
-	/* Don't update negative cache when attempting to re-connect */
-	if (ret && !(ap->flags & MOUNT_FLAG_REMOUNT)) {
+	if (ret) {
 		time_t now = time(NULL);
 		int rv = CHE_OK;
